pkg/cleanup/services.go

```diff
@@ -23,7 +24,7 @@ func (s *services) Run() error {
 	var msg []string
 
 	for _, role := range []string{"controller", "worker"} {
-		if err := install.UninstallService(role); err != nil && !errors.Is(err, fs.ErrNotExist) {
+		if err := install.UninstallService(role); err != nil && !errors.Is(err, fs.ErrNotExist) && !isExitCode(err, 1) {
```

Comment:
I wonder if `fs.ErrNotExist` can come with code 1, which would make this check fail 🤔

Maybe:

```suggestion
		if err := install.UninstallService(role); err != nil && !(errors.Is(err, fs.ErrNotExist) || isExitCode(err, 1)) {
```

Reply:
I think `!foo && !bar` and `!(foo || bar)` are exactly the same thing?

But I liked your variant better.